scripts/chroma-test-helpers/create-and-check-chroma.sh

```diff
@@ -3,13 +3,4 @@
 set -e
 
 # $1 is the path for chroma to write the contents to. The symbol "&" runs process in background
-echo "Current venv is:"
-echo "$VIRTUAL_ENV"
-echo "Current path is:"
-echo "$PATH"
-ls -l "$VIRTUAL_ENV/bin/chroma"
-echo "================"
-cat "$VIRTUAL_ENV/bin/chroma"
-echo "================"
-# chroma run --path "$1" &
```

Comment:
Removing these diagnostic "print" statements added during troubleshooting CI. Didn't want to trigger a whole new CI run just to take these out on the last PR.

Reply:
@Coniferish the gist of this change is pretty simple:
- Move `unstructured/partition/common.py` into a subdirectory `unstructured/partition/common/`
- Extract the metadata-related functions from `common.py` into `metadata.py`

All the rest is updating import statements. No code changes.